src/index.ts

```diff
@@ -2475,6 +2476,316 @@ function createServer({ config }: { config?: any } = {}) {
     })
   )
 
+  // Prompts tool for MCP clients  // Import prompts as an ES module
+    // Register the list_prompts tool
+  server.tool(
+    "list_prompts",
+    "List available LLM prompt templates for common HubSpot workflows.",
+    {},
+    (_args, _extra) => ({
+      content: [
+        {
+          type: "text" as const,
+          text: JSON.stringify(prompts, null, 2)
+        }
+      ]
+    })
+  );
```

Comment:
`listTools()` is actually a feature of MCP (see [Discovering Prompts](https://modelcontextprotocol.io/docs/concepts/prompts#discovering-prompts)) so we can remove this:
```suggestion
```